network/engine.go

```diff
@@ -12,22 +12,22 @@ import (
 type Engine interface {
 
 	// SubmitLocal submits an event originating on the local node.
-	SubmitLocal(event interface{})
+	SubmitLocal(channel Channel, event interface{})
 
 	// Submit submits the given event from the node with the given origin ID
 	// for processing in a non-blocking manner. It returns instantly and logs
 	// a potential processing error internally when done.
-	Submit(originID flow.Identifier, event interface{})
+	Submit(channel Channel, originID flow.Identifier, event interface{})
 
 	// ProcessLocal processes an event originating on the local node.
-	ProcessLocal(event interface{}) error
+	ProcessLocal(channel Channel, event interface{}) error
 
 	// Process processes the given event from the node with the given origin ID
 	// in a blocking manner. It returns the potential processing error when
 	// done.
-	Process(originID flow.Identifier, event interface{}) error
+	Process(channel Channel, originID flow.Identifier, event interface{}) error
 }
 
 type MessageProcessor interface {
-	Process(originID flow.Identifier, message interface{}) error
+	Process(channel Channel, originID flow.Identifier, message interface{}) error
```

Comment:
> BTW, the networking layer should hopefully soon use a more generic interface, as described in this Flip: https://github.com/onflow/flow/blob/alex/networking-api-flip/flips/20210201-consuming-messages-from-network-layer.md I think it would be reasonable to add the `channel name` to the `Consume` method.

@AlexHentschel are you referring to this?

Reply:
yes 👍

Reply:
Yes, and I've updated the PR to remove channel argument from `SubmitLocal` and `ProcessLocal` functions. It's only in `Process` and `Submit` now.

Reply:
So Submit, and SubmitLocal have been deprecated already (line 11: 

> // TODO: DEPRECATED replace with MessageProcessor

but there are still some instances of it being used.